DSharpPlus.CommandsNext/CommandsNextExtension.cs

```diff
@@ -692,9 +684,12 @@ public async Task DefaultHelpAsync(CommandContext ctx, [Description("Command to
                             break;
                         }
 
-                        cmd = ctx.Config.CaseSensitive
-                            ? searchIn.FirstOrDefault(xc => xc.Name == c || (xc.Aliases != null && xc.Aliases.Contains(c)))
-                            : searchIn.FirstOrDefault(xc => xc.Name.ToLowerInvariant() == c.ToLowerInvariant() || (xc.Aliases != null && xc.Aliases.Select(xs => xs.ToLowerInvariant()).Contains(c.ToLowerInvariant())));
+                        var (comparison, comparer) = ctx.Config.CaseSensitive switch
+                        {
+                            true  => (StringComparison.InvariantCulture, StringComparer.InvariantCulture),
+                            false => (StringComparison.InvariantCultureIgnoreCase, StringComparer.InvariantCultureIgnoreCase)
+                        };
+                        cmd = searchIn.FirstOrDefault(xc => xc.Name.Equals(c, comparison) || (xc.Aliases != null && xc.Aliases.Contains(c, comparer)));
```

Comment:
For this one we will need to invoke help for command, ideally with aliases.
`d# help setprefix`
`d# help channelprefix`
`d# help channelPreFix`
In this case screenshot of code will be too large, so i chosen one with "most specific case"
![image](https://user-images.githubusercontent.com/50947930/152696950-fcd354ee-69fb-47e9-81da-42cc6c6ac865.png)